x-pack/plugins/task_manager/server/queries/mark_available_tasks_as_claimed.test.ts

```diff
@@ -18,7 +19,17 @@ import {
 import { TaskTypeDictionary } from '../task_type_dictionary';
 import { mockLogger } from '../test_utils';
 
+let clock: sinon.SinonFakeTimers;
+
 describe('mark_available_tasks_as_claimed', () => {
+  beforeEach(() => {
+    clock = sinon.useFakeTimers();
```

Comment:
This is interesting doesn't jest.useFakeTimers() work?

Reply:
I'm sure that would have worked as well :)